packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -0,0 +1,159 @@
+
+Sample usage, composes `Accordion` and `AccordionItem`:
+
+```jsx
+import { Accordion, AccordionItem } from '@razorpay/blade';
+
+<Accordion variant="ordered" defaultIndex={0}>
+  <AccordionItem
+    title="Can I create linked accounts using Route?"
+    description="You can use Razorpay Route from the Dashboard or using APIs to transfer money to customers."
+  >
+    <div>Slot</div>
+  </AccordionItem>
+  <AccordionItem title="How can I transfer money to customers?" description="Just use Razorpay" />
+</Accordion>;
+```
+
+### Accordion
+
+| Prop         | Type                   | Default     | Description                                                    | Required |
+| ------------ | ---------------------- | ----------- | -------------------------------------------------------------- | -------- |
+| variant      | `unordered`, `ordered` | `unordered` | `ordered` shows a numeric prefix to titles                     |          |
+| defaultIndex | `number`               | `undefined` | Makes the passed item index expanded by default (uncontrolled) |          |
+| index        | `number`               | `undefined` | Expands the passed index (controlled)                          |          |
+| onChange     | `function`             | `undefined` | Callback for change in any item's expanded state (controlled)  |          |
+
+> **Note**
+>
+> - Also includes layout based styling props
+> - By default the accordion renders in all items collapsed state, at max only one item can be expanded at a time (design restriction)
+> - `defaultIndex` can accept an accordion item index to render the accordion in an uncontrolled state
+> - For using accordion in controlled state, use `index` coupled with `onChange`
+> - `onChange` callback signature `({ index }) => {}`, `index` represents the expanded item's index
+
+### AccordionItem
+
+| Key         | Type            | Default     | Description                                                         | Required |
+| ----------- | --------------- | ----------- | ------------------------------------------------------------------- | -------- |
+| title       | `string`        | `undefined` | Title text content                                                  | ✅       |
+| description | `string`        | `undefined` | Body text content                                                   |          |
+| icon        | `IconComponent` | `undefined` | Renders a Blade icon as title prefix (requires `unordered` variant) |          |
+| children    | `JSX`           | `undefined` | Slot, renders any custom content                                    |          |
+
+## a11y
+
+Web:
+
+Follows the standard practices (aria expanded, controls, etc.) [ARIA guide](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/), [example](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/#rps_label):
+
+- <kbd>Enter</kbd> or <kbd>Space</kbd> can toggle accordion
+- <kbd>Tab</kbd> key to cycle focus
```

Comment:
will the tab will trap focus? like you start at first item and then when you tab tab tab and after you reach the last item shall it go to next focussable element or go to first item in the group again?